webapp/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "private": true,
+  "type": "commonjs",
```

Comment:
Wondering if this change is required

Reply:
Might not be required, I was working to clean up some of the webpack config imports and such and that may be leftover from that. We can remove it if you'd like!